docs/topics/vertx-community-and-product-versions.adoc

```diff
@@ -1,15 +1,14 @@
 [[vertx-community-and-product-versions]]
 = Community and Product Versions of {Vertx}
 
-.Community {Vertx} Version
-// link TBD
-Some of the {Vertx} components included in this release are not fully tested to run on OpenShift, and are provided as a xref:vertx-tech-preview-components[limited-supportability Technology Preview].
-These components are based on the community version `3.4.2` of {Vertx}.
+.Supported {Vertx} Version
 
-.Certified {Vertx} Version
+{Vertx} components version `3.4.2.redhat-006` are supported and provided as part of a Red Hat subscription.
+Supported {Vertx} runtime artifacts are available in the link:https://maven.repository.redhat.com/ga/[Red Hat Middleware JBoss General Availability Maven Repository^].
 
-The productized {Vertx} version `3.4.2.redhat-006` is certified to run on OpenShift and provided as part of the Red Hat subscription.
-The certified {Vertx} runtime artifacts are available from the link:https://maven.repository.redhat.com/ga/[Red Hat Middleware JBoss General Availability Maven Repository^].
+.Technology Preview {Vertx} Components
```

Comment:
@Ladicek I have included a separate paragraph about tech preview  to set it apart form the Supported artifacts. WDYT?